src/timer/Timer.js

```diff
@@ -1,17 +1,21 @@
-function Timer(seconds) {
-  this.seconds = seconds;
-}
+// Refactor to use ES6 classes
+module.exports = class Timer {
+  constructor(seconds) {
+    this.seconds = seconds;
+  }
 
-Timer.prototype.start = function() {
-  var instance = this;
-  var timerInterval = setInterval(function() {
-    if (instance.seconds === 0) {
-      clearInterval(timerInterval);
-    }
+  start() {
+    this.timer = setInterval(() => {
+      if (this.seconds === 0) {
+      // eslint-disable-next-line no-console
+        console.log('Done!');
+        clearInterval(this.timer);
+        return;
+      }
 
-    console.log(instance.seconds);
-    instance.seconds -= 1;
-  }, 1000);
+      // eslint-disable-next-line no-console
```

Comment:
Nice use of the eslint-disable comments. They're very handy for ignoring rules in special cases.